src/track/external-captions-handler.js

```diff
@@ -0,0 +1,380 @@
+import Error from '../error/error'
+import * as Utils from '../utils/util'
+import {Parser, StringDecoder} from './text-track-display'
+import {TrackType} from './track-type';
+import TextTrack from './text-track';
+import {CustomEventType, Html5EventType} from '../event/event-type';
+import FakeEvent from '../event/fake-event';
+import getLogger from '../utils/logger';
+import EventManager from '../event/event-manager';
+import FakeEventTarget from '../event/fake-event-target';
+
+
+const cuesStatus = {
```

Comment:
Convention for enums ->`CuesStatus`

Also annotate this const.

Reply:
changed